docker/volumeplugin/exec.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const (
 	statusSuccess      = "Success"
-	statusFailure      = "Failed"
```

Comment:
This is in fact correct.  These constants were copied from k8s https://github.com/kubernetes/kubernetes/blob/b359034817685a8d25bb51bae765308d9d200da0/pkg/volume/flexvolume/flexvolume_util.go#L45